apps/fhir/server/tests/test_authentication.py

```diff
@@ -189,38 +189,42 @@ def test_match_fhir_id_malformed_mbi(self):
-        '''
-        with HTTMock(self.create_fhir_mock(self.LYING_KEY, self.NOT_FOUND_KEY)):
-            with self.assertRaisesRegexp(UpstreamServerException, "^Duplicate.*"):
-                fhir_id, hash_lookup_type = match_fhir_id(
-                    mbi=self.test_mbi,
-                    mbi_hash=self.test_mbi_hash,
-                    hicn_hash=self.test_hicn_hash, request=self.request)
-
-    def test_match_fhir_id_lying_mbi(self):
-        '''
-            Testing responses: HICN = success
-                               MBI = lying
-            Expecting: UpstreamServerException exception raised
-            Note: lying means response total=1, but there are multiple
-                  Patient resources in the response.
-        '''
-        with HTTMock(self.create_fhir_mock(self.SUCCESS_KEY, self.LYING_KEY)):
-            with self.assertRaisesRegexp(UpstreamServerException, "^Duplicate.*"):
-                fhir_id, hash_lookup_type = match_fhir_id(
-                    mbi=self.test_mbi,
-                    mbi_hash=self.test_mbi_hash,
-                    hicn_hash=self.test_hicn_hash, request=self.request)
+    # BB2-3696 remove reference to FHIR search result Bundle.total
+    # due to Bundle.total removal, this test no longer applicable
+    # def test_match_fhir_id_lying_hicn(self):
+    #     '''
+    #         Testing responses: HICN = lying
+    #                            MBI = not_found
+    #         Expecting: UpstreamServerException exception raised
+    #         Note: lying means response total=1, but there are multiple
+    #               Patient resources in the response.
+    #     '''
+    #     with HTTMock(self.create_fhir_mock(self.LYING_KEY, self.NOT_FOUND_KEY)):
+    #         with self.assertRaisesRegexp(UpstreamServerException, "^Duplicate.*"):
+    #             fhir_id, hash_lookup_type = match_fhir_id(
+    #                 mbi=self.test_mbi,
+    #                 mbi_hash=self.test_mbi_hash,
+    #                 hicn_hash=self.test_hicn_hash, request=self.request)
+
+    # BB2-3696 remove reference to FHIR search result Bundle.total
+    # due to Bundle.total removal, this test no longer applicable
+    # def test_match_fhir_id_lying_mbi(self):
+    #     '''
+    #         Testing responses: HICN = success
+    #                            MBI = lying
+    #         Expecting: UpstreamServerException exception raised
+    #         Note: lying means response total=1, but there are multiple
+    #               Patient resources in the response.
+    #     '''
+    #     with HTTMock(self.create_fhir_mock(self.SUCCESS_KEY, self.LYING_KEY)):
+    #         with self.assertRaisesRegexp(UpstreamServerException, "^Duplicate.*"):
+    #             fhir_id, hash_lookup_type = match_fhir_id(
+    #                 mbi=self.test_mbi,
+    #                 mbi_hash=self.test_mbi_hash,
+    #                 hicn_hash=self.test_hicn_hash, request=self.request)
```

Comment:
Thoughts on removing this? Not sure if this will be useful moving forward to leave these commented out.